src/main/resources/application-test.yaml

```diff
@@ -1,10 +1,11 @@
 spring:
   datasource:
-    url: jdbc:postgresql://postgres:5432/${DB_NAME}
+    url: jdbc:tc:postgresql:12.15:///${DB_NAME}
```

Comment:
test properties are using for test environment, not for tests

all properties for integration tests should be in test/resources folder